chanzuckerberg/idseq-web
# Yf/lineage consolidation

Test plan: 

* run docker-compose run web "rake load_lineage_db" 
* check the taxon_lineages table with all names populated.
